.github/workflows/data_integrity_spec_test.yaml

```diff
@@ -0,0 +1,44 @@
+name: Data Integrity Spec Test
+# Tests the VC Model Spec against the vc-test-suite - https://github.com/w3c/vc-test-suite
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: ./.github/actions/rust-setup
+        name: Setup rust toolchain
+
+      - uses: knox-networks/github-actions-public/.github/actions/setup-protofetch@main
+        with:
+          cross-repo-username: "developersKnox"
+          cross-repo-token: ${{ secrets.PROTOFETCH_GITHUB_TOKEN }}
+
+      - name: Test protos
+        run: (cd registry_resolver; protofetch fetch) && git diff --exit-code
+
+      - name: Build CLI
+        run: cargo build --package cli --release --features="v2_test"
+
+      - name: Add CLI to bin
+        run: sudo cp $PWD/target/release/ssi_cli /bin/ssi_cli
```

Comment:
What if we skipped this step and just ran `ssi_cli` from this `$PWD/target/release/ssi_cli` path in the step below?

Reply:
@gshuflin For this it might be possible but I'd prefer to follow the same model as the other spec test (which did require it to be part of the bin folder